Processing debug_priv.h causes doxygen 1.11.0 to encounter an assertion fail, aborting the build:

/usr/include/c++/13.2.1/string_view:256: constexpr const std::basic_string_view<_CharT, _Traits>::value_type& std::basic_string_view<_CharT, _Traits>::operator[](size_type) const [with _CharT = char; _Traits = std::char_traits<char>; const_reference = const char&; size_type = long unsigned int]: Assertion '__pos < this->_M_len' failed.


--- a/docs/Doxyfile.in
+++ b/docs/Doxyfile.in
@@ -550,7 +550,7 @@
 # excluded from the INPUT source files. This way you can easily exclude a 
 # subdirectory from a directory tree whose root is specified with the INPUT tag.
 
-EXCLUDE                = ../config.h
+EXCLUDE                = ../config.h ../../src/common/debug_priv.h
 
 # The EXCLUDE_SYMLINKS tag can be used select whether or not files or 
 # directories that are symbolic links (a Unix filesystem feature) are excluded 
